addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -155,6 +148,14 @@ export class HtmlField extends Component {
                 }
             })();
         });
+        onMounted(() => {
+            const $editable = this.wysiwyg && this.wysiwyg.getEditable();
+            if ($editable && $editable.length) {
+                $editable.off('.checklistBinding');
+                $editable.on('mousedown.checklistBinding', 'ul.o_checklist > li', this._onReadonlyClickChecklist.bind(this));
```

Comment:
Why did you change from click to mousedown?

Reply:
I called the `mousedown` event to ensure that it is triggered before the editing value is updated.

Reply:
I guess that's an ok tradeoff.